### ISSUE TITLE
src/Snap/Internal/Http/Server/Socket.hs:176:5: error:     Conflicting definitions for ‘acquire’

### ISSUE THREAD

**Issue by l29ah** [snapframework/snap-server#121]
Building 1.1.1.0:
```
[19 of 20] Compiling Snap.Internal.Http.Server.Socket ( src/Snap/Internal/Http/Server/Socket.hs, dist/build/Snap/Internal/Http/Server/Socket.o )

src/Snap/Internal/Http/Server/Socket.hs:176:5: error:
    Conflicting definitions for ‘acquire’
    Bound at: src/Snap/Internal/Http/Server/Socket.hs:176:5-11
              src/Snap/Internal/Http/Server/Socket.hs:182:5-11
    |
176 |     acquire   = openFd fPath ReadOnly Nothing defaultFileFlags
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
```

**Comment by mightybyte** [comment]
I just merged PR #120 which should fix this.  Can you see if that works for you?

**Comment by l29ah** [comment]
It works, thank you.
